evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -123,7 +124,7 @@ def test_make_pipeline_all_nan_no_categoricals(input_type, problem_type):
         y = pd.Series([0, 2, 1, 2])
 
     for estimator_class in estimators:
-        for problem_type in estimator_class.supported_problem_types:
```

Comment:
This is no longer valid because if the fixture `problem_type` was time series then `pipeline_class = _get_pipeline_base_class(problem_type)` will now have a type of `TimeSeriesPipelineBaseMeta `. If the `problem_type` in this for loop (duplicate names, yikes) is binary, then pipeline will be of type PipelineBaseMeta and the `assert isinstance(pipeline, type(pipeline_class))` line will fail.

Reply:
These tests still iterate through all problem types and their estimators right? If so this should be good.

Reply:
Si, si, fixture goes through all of problem types and then we get all the estimators for the problem type!